mlflow/store/abstract_store.py

```diff
@@ -50,6 +50,16 @@ def get_experiment(self, experiment_id):
         """
         pass
 
+    @abstractmethod
+    def get_experiment_by_name(self, experiment_name):
+        """
+        Fetches the experiment by name from backend store.
+        Throws an exception if experiment is not found or permanently deleted.
```

Comment:
The FileStore and RestStore implementations of this method do not throw an exception in the case that the experiment is not found; they return `None`.

Reply:
Good catch! Thanks..